saleor/plugins/webhook/plugin.py

```diff
@@ -728,11 +728,38 @@ def gift_card_export_completed(
             export,
         )
 
+    def _get_webhooks_for_order_events(
+        self,
+        event_type: str,
+        order: "Order",
+        webhooks: Optional[Iterable["Webhook"]] = None,
+    ) -> Iterable["Webhook"]:
+        """Get webhooks for order events.
+
+        Fetch all valid webhooks and filter out the ones that have a subscription query
+        with filter that doesn't match to the order.
+        """
+        order_channel_slug = order.channel.slug
```

Comment:
Does the Saleor make an additional query? Maybe we could pass the channel here for outside code?